### PR TITLE
Exclude unpublished

### PR DESCRIPTION
**Pull request by markcmurphy**

### CODE REVIEW

File: .stoplight.json [hunk 1]
```diff
@@ -17,5 +17,5 @@
         "include": ["**"]
       }
     },
-    "exclude": ["/README.md"]
+    "exclude": ["/README.md", "reference/pages.v3.yml", "CONTRIBUTING.md"]
```
Comment (slsriehl) [inline .stoplight.json:20]:
```suggestion
        "exclude": [
          "/README.md",
          "reference/pages.v3.yml",
          "CONTRIBUTING.md"
        ]
```

File: reference/carts.sf.yml [hunk 2]
```diff
@@ -668,7 +678,7 @@ components:
       properties:
         amount:
           type: number
-          description: Value must be between 1.00 and 1,000.00 in the store's default currency.
+          description: 'Value must be between 1.00 and 1,000.00 in the store''s default currency.'
```
Comment (slsriehl) [inline reference/carts.sf.yml:681]:
```suggestion
          description: 'Value must be between 1.00 and 1,000.00 in the storeʼs default currency.'
```

**Comment by slsriehl** [comment]
- re: customers_v2 vs customers.v2 -- well-spotted.  @markcmurphy did you check the commit history to see if there's some complex change stuff to suss out stemming from there having been two files in the mix (or pending tickets we can mark irrelevant)?
- didn't verify that the hidden endpoints correspond to those located in Friday's work
- made typographic updates
- can approve first thing tomorrow if needed -- let me know.

**Comment by markcmurphy** [comment]
> 

I took a look and the customers_v2 file didn't seem to have any meaningful updates. It was made as a test at some point and has just been hanging out.

**Pull request merged**